libvast/vast/concept/parseable/vast/schema.hpp

```diff
@@ -182,13 +182,13 @@ struct symbol_table_parser : parser<symbol_table_parser> {
 };
 
 template <>
-struct parser_registry<symbol_table> {
+struct parser_registry<symbol_map> {
   using type = symbol_table_parser;
 };
 
 namespace parsers {
 
-constexpr auto symbol_table = symbol_table_parser{};
+constexpr auto symbol_map = symbol_table_parser{};
```

Comment:
I don't understand why this was renamed. Please undo, or rename the parser accordingly.

Reply:
The reason is that @tobim removed a class called `symbol_table`, which was part of the parser framework. I had to undo this because I'm actually using the `symbol_table` parser here internally. @tobim's `symbol_table_parser` is strictly speaking also not a "classic" symbol table, which is why I renamed it.

@tobim how do you feel about the renaming?

Reply:
I'm fine with the parser itself being named `symbol_map_parser` as well, e.g., either of these options:
- `parsers::symbol_table` and `symbol_table_parser`
- `parsers::symbol_map` and `symbol_map_parser`

Reply:
Note that we have this now:

```cpp
using symbol_map = std::unordered_map<std::string, type>;
```

This type was called `symbol_table` before because the old symbol table parser class with the same name go removed. So now we're back to the old state, which works just fine in my opinion.

Reply:
I'm not talking about `vast::symbol_map`, but rather about `parsers::symbol_map`.

For all other parsers, it's `X_parser` and `parsers::X`. That's not the case here.

Reply:
You're right. Somehow we were barking up the wrong tree. 🐶